### ISSUE TITLE
The added errors in the onValidation event do not highlight the fields

### ISSUE THREAD

**Issue by DartVerder** [Haulmont/jmix-core#236]
**Environment:**
Jmix platform version: 1.0.2

**Steps:**
1. Create entity
2. Create several basic type fields
3. Create browser and editor screens
4. On editor screen generate handler "ValidationEvent"
5. Inject one of the fields to the screen
6. Add error to the validation event
```
ValidationErrors validationErrors = new ValidationErrors();
validationErrors.add(emailField,"That's wrong");
event.addErrors(validationErrors);
```
7. Run application
8. Create an instance of the entity 
9. Click ok

**AR:** the error field is not highlighted. An error message appears
**ER:** the error field is highlighted in red
Note: The same behaviour after overriding validateAdditionalRules method

**Comment by glebfox** [comment]
We don't have API to make fields highlighted in red, i.e. mark as invalid. **ValidateAdditionalRules (ValidationEvent)** functionality is similar to cross field validation, i.e. it checks dependency between several fields and is performed only on entity commit. If we mark a field as invalid (somehow), the user expects the error to disappear once the field value is changed, but it isn't feasible because `ValidationEvent` is fired on entity commit. 

Long story short, we add a field to `validationErrors` just to focus it as a hint where to start fix values.
